apps/extension/src/ui/domains/Staking/Bittensor/utils/constants.ts

```diff
@@ -10,3 +12,5 @@ export const TALISMAN_FEE_RECEIVER_ADDRESS_BITTENSOR =
   "5DzsVV2L4M9r4uWoyarzPyhfeCv6DDAEs5rM2bpHjmerPcGa"
 
 export const TALISMAN_FEE_BITTENSOR = 0.3
+
+export const DEFAULT_ROOT_CLAIM_TYPE: RootClaimType = "Keep"
```

Comment:
The on-chain default is still `Swap`, eventhough they are planning to change it to `Keep` soon

```suggestion
export const DEFAULT_ROOT_CLAIM_TYPE: RootClaimType = "Swap"
```